source.txt

```diff
@@ -354,12 +354,12 @@ the document. If this software is integrated with Alice's addressbook
 software, then the addressbook entry for Bob may also be updated with
 the authentication methods for which Alice provided the identifier,
 those which were discovered programmatically during the process, and the
-one that Bob uiltimately used to authenticate.
+one that Bob ultimately used to authenticate.
 
 The access page MAY also give human-readable instructions to Bob, like
 "please don't share this document publically", "please don't share
 outside our Petanque team", or "please discard after reading, or after
-30 days". Whether or not Bob follows such instructions is out of scope.
+30 days". How Bob enforces such rules is out of scope.
```

Comment:
Once Bob has (read-)access to the document, he can't enforce rules upon himself, because he's a human actor with free choice to follow rules or not. I am also uncomfortable with mentioning or hinting at DRM, for obvious political reasons. ;)

I think the only thing we could add here is that Alice's application may track if the document has already been retrieved with Bob's credentials or not. After that, Bob is free to follow (enforce upon himself) Alice's instructions or not, right?